apps/prairielearn/src/pages/instructorIssues/instructorIssues.ejs

```diff
@@ -83,7 +83,7 @@
                     <td>
                       Shows all issues that were reported by a user ID like <code>USER</code>.
                       <br>
-                      <strong>Example:</strong> <code>user:nwalter2@illinois.edu</code> shows all issues that were reported by <code>nwalter2@illinois.edu</code>.</td>
+                      <strong>Example:</strong> <code>user:student@example.edu</code> shows all issues that were reported by <code>student@example.edu</code>.</td>
```

Comment:
Do we want to be RFC 2606 compliant and use `example.com` instead? (we are doing this in PT, fwiw)

Reply:
Sure! example.edu resolves to a page that looks exactly like example.com, so I didn't even realize it wasn't an "official" example domain.

Reply:
Done in 9bad16a.